src/lib/trakeva.ml

```diff
@@ -133,7 +94,7 @@ module type KEY_VALUE_STORE = sig
       Deferred_result.t
   (** Get all the keys in a given collection as a list. *)
```

Comment:
Shouldn't we remove the documentation from the `.ml` copy of the module type?
I don't like having that duplication Vs future edits.

(i'm commenting here because github can't comment outside the PR's diff...)